plugin/lighthouse/parse.go

```diff
@@ -0,0 +1,117 @@
+package lighthouse
```

Comment:
might we worth looking at importing this to be used as an external dependency. As these methods and functions are not an exported part of the package this basically means you'll have to create a PR or raise an Issue and make your case for them to export it. I guess it won't happen though but might worth the try.

NOT REVIEWED